client/task-list/tasks/payments/methods.js

```diff
@@ -79,100 +79,9 @@ export function getPaymentMethods( {
 		wcPayIsConnected = false,
 	} = onboardingStatus;
 
-	const hasCbdIndustry =
```

Comment:
I've just tried to do as much tidy up as I can in this area, removed Fragment, removed lodash that wasn't really needed, cleaned up the array manipulation and renamed the logos to reflect that they're not icons.